app/src/main/java/woowacourse/movie/domain/MovieSchedule.kt

```diff
@@ -0,0 +1,54 @@
+package woowacourse.movie.domain
+
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+
+class MovieSchedule(private val startDate: LocalDate, private val endDate: LocalDate) {
+
+    fun getScheduleDates(): List<String> = getDatesBetweenTwoDates().map {
+        it.format(dateTimeFormatter)
+    }
+
+    private fun getDatesBetweenTwoDates(): List<LocalDate> {
+        val numOfDaysBetween = ChronoUnit.DAYS.between(startDate, endDate)
+
+        return (0..numOfDaysBetween).map {
+            startDate.plusDays(it)
+        }.toList()
```

Comment:
```suggestion
        return (0..numOfDaysBetween).map {
            startDate.plusDays(it)
        }
```
filter, map과 같은 컬렉션 함수들은 기본적으로 새로운 컬랙션을 만들어줍니다.